ResourceManager/GitPluginBase.cs

```diff
@@ -30,7 +30,6 @@ protected GitPluginBase()
         protected void SetNameAndDescription(string name)
         {
             Name = name;
-            Description = name;
```

Comment:
I think it should be left for the settings. As a non-localized key. For legacy plugins to work.